build.sbt

```diff
@@ -9,6 +9,9 @@ val isScala3 = Def.setting(
   CrossVersion.partialVersion(scalaVersion.value).exists(_._1 == 3)
 )
 
+ThisBuild / scalaVersion       := Scala3
```

Comment:
This had to be done for technical sbt reasons, basically sbt-github-actions will only pick up these values if they are in the `ThisBuild` scope. In general the concept of `commonSettings` is outdated for newer sbt builds (thats why `ThisBuild` was invented) but changing the entire `build.sbt` would be for another PR.

Reply:
> In general the concept of commonSettings is outdated for newer sbt builds...

Ok, thank you! It seems I'm a bit out of touch with the state-of-the-art.
It's time I look a bit through more modern sources to see how things are done nowadays.
Therefore your comment are very much appreciated, thank you again!